cli/command/image/prune.go

```diff
@@ -42,6 +43,7 @@ func NewPruneCommand(dockerCli *command.DockerCli) *cobra.Command {
 	flags := cmd.Flags()
 	flags.BoolVarP(&opts.force, "force", "f", false, "Do not prompt for confirmation")
 	flags.BoolVarP(&opts.all, "all", "a", false, "Remove all unused images, not just dangling ones")
+	flags.BoolVarP(&opts.keepDownloadCache, "keep-download-cache", "k", false, "Don't clean out the download cache")
```

Comment:
I find boolean flags that default to true confusing, but I'll change it if anyone has a strong opinion.